frontend/src/metabase/search/types.ts

```diff
@@ -18,7 +17,7 @@ export interface WrappedResult extends SearchResult {
     width?: number;
     height?: number;
   };
-  getCollection: () => Partial<Collection>;
+  getCollection: () => SearchResult["collection"];
```

Comment:
Also using `Partial` here causes type errors, since the functions that use the collection expect some keys to definitely be in the collection, so typing with `Partial`  just doesn't work